server/src/data/models/abstract/abstract.model.js

```diff
@@ -1,6 +1,6 @@
 import { Model } from 'objection';
 
-class Abstract extends Model {
+class AbstractModel extends Model {
```

Comment:
we already have `model` in the file name. So we need to add it to the class name. Rename it as { Abstract as AbstractMobel } in places where you need it. Should be fixed for all models

Reply:
@what1s1ove you mean rename in current file like on screen or at import in different modules?
![image](https://user-images.githubusercontent.com/43109766/168745189-66f7c076-6b3b-4fb0-9c8d-180f3c65580a.png)

Reply:
**at import in different modules** 👍